variables.tf

```diff
@@ -75,6 +75,19 @@ variable "cos_tags" {
   default     = []
 }
 
+variable "access_tags" {
+  type        = list(string)
+  description = "A list of access tags to apply to the resources created by the module, see https://cloud.ibm.com/docs/account?topic=account-access-tags-tutorial for more details"
```

Comment:
Probably need to be more specific. Are they applied to the COS instance? The bucket? Both? If both, do we need separate variables (not sure on this tbh)